components/Command/Command.module.css

```diff
@@ -75,6 +75,10 @@
   }
 }
 
+.command .line div {
+  margin-right: var(--m-1-5);
+}
+
```

Comment:
I think we'll need something like this to handle plain text as well. Getting some margin cutoff in plain text visible here. (the `quay` cutoff on bottom line). 
Adding `margin-right: var(--m-1-5)` for the `.wrapper, .label` selector seemed to do the trick without messing with code blocks since `.Command_line` takes priority but, if we removed it from there then just handling it in the wrapper label would work everywhere? Maybe I missed a spot! 
<img width="1114" alt="Screen Shot 2022-06-27 at 1 53 49 PM" src="https://user-images.githubusercontent.com/5201977/176024454-dd829361-6c79-442d-8526-a4753a8407c7.png">

Reply:
Good catch! Thank you